src/js/controller/captions.js

```diff
@@ -1,7 +1,9 @@
 define(['utils/helpers',
     'utils/render-captions-natively',
-    'controller/tracks-loader'
-], function(utils, renderCaptionsNatively, tracksLoader) {
+    'controller/tracks-loader',
+    'utils/track/create-id',
+    'utils/track/create-label'
+], function(utils, renderCaptionsNatively, tracksLoader, createTrackId, createTrackLabel) {
```

Comment:
I think it now makes sense to put `renderCaptionsNatively` `createTrackId` `createTrackId` in one file called `captionUtils.js`. And to unit test them too. 😸